data/hai/hai reveal 1 intro.txt

```diff
@@ -11,48 +11,675 @@
+			`	After a short while you receive a message requesting that you report to the operations center inside the main promenade. You proceed to equip your own suit and exit your ship. The station itself is well preserved, except for the lack of atmosphere. Nothing seems to have vented it, no big hole in the ceiling, and the expansive ceiling windows of the main promenade provide a crystal clear view of the station's many decks towering above you. Exterior navigation lights are gradually flicking on up the height of the station, so that walking through feels like watching a ghost city slowly come to glittering life. Emergency floor lighting has been activated, showing you the way.`
+			`	Once through the operation center's security airlock you are able to remove your helmet. Kelamursh's subordinate, who briefed you in the offices and appears to be in charge in this area, swiftly approaches from the other side. "Captain <last>! It is splendid you could make it here so quickly. We have begun the re-powering process, but it will take time and supplies to re-pressurize more than the docking and operations areas. These stations are not small."`
+			`	He chuckles as if having made a joke, as if you hadn't just walked beneath the open view of over three hundred decks above you. He seems oblivious to the strangeness of this, so perhaps it is a Hai thing.`
+			`	"I am sure you have questions but they will have to wait until we're more established. We have supplies coming and they will be vulnerable to any pirates that may poke their noses in. We need you to get out there and ensure the supplies can all arrive here safely. Local security is still about, so it shouldn't be too problematic."`
+			`	He collects a device from a carry crate and hands it to you. "Now that we've got the basics up and running, the station is in safe mode. You'll need to plug this into your communications system to re-dock safely. The supply convoy has them pre-installed."`
+			choice
+				`	(Go immediately.)`
+					accept
+				`	(Loiter to find out more.)`
+					goto loiter
+				`	"What were we getting paid for this?"`
+			`	He frowns. "It was based on your ship's combat quality. I don't have that tablet with me right now though. Once the supply ships have arrived I should have access to that again. Now go on, you won't get paid at all if the supply ships don't make it."`
+				accept
+			label loiter
+			`	He blinks at you. "Well, what are you waiting for? Go! Come back when all the supply ships have arrived safely."`
+			`	You are left with little choice other than to nod and get on your way.`
+				accept
+	on accept
+		set "hr: sandsnap project"
+	on visit
+		dialog "The supply ships have not all arrived safely yet."
```

Comment:
```suggestion
		dialog "The supply ships have not all arrived safely yet. Depart and ensure they are in the <system> system."
```
Hmm, I'm not entirely sure if this will display the right thing since the mission has no destination.

I could get an `<origin system>` substitution, though...